### PR TITLE
Feature/includes dir init

### PR DESCRIPTION
**Pull request by ilya-shaydullin**
Added register logic for partials path (_includes dir) in shaper core. 
Updated README.

### CODE REVIEW

File: shaper-core/src/main/kotlin/dev/icerock/tools/shaper/core/HandlebarsFactory.kt [hunk 1]
```diff
@@ -6,10 +6,16 @@ package dev.icerock.tools.shaper.core
...
+import java.io.File
+import java.nio.file.Paths
 
 object HandlebarsFactory {
-    fun create(): Handlebars {
-        val handlebars = Handlebars()
+    fun create(includePath: String = "_includes"): Handlebars {
+        val partialDir = File("${Paths.get("").toAbsolutePath()}/$includePath")
+        partialDir.mkdirs()
+        val handlebars = Handlebars(FileTemplateLoader(partialDir, ".hbs"))
```
Comment (Alex009) [inline shaper-core/src/main/kotlin/dev/icerock/tools/shaper/core/HandlebarsFactory.kt]: will be this work if includePath was not in work directory? we support search of template in multiple templates directories at now and this should work with any template dir

File: shaper-core/src/test/resources/Test.kt [hunk 2]
```diff
@@ -1,4 +1,8 @@
 package dev.icerock.test
 
 class Test {
+    fun hello() {
+        println("Hi Developer!")
+        println("Test")
+    }
 }
```
Comment (Alex009) [inline shaper-core/src/test/resources/Test.kt:8]: this should be removed :)

File: shaper-core/src/test/kotlin/dev/icerock/tools/shaper/core/ShaperTest.kt [hunk 3]
```diff
@@ -25,6 +25,7 @@ class ShaperTest {
         val config = TemplateConfig(
             globalParams = mapOf("packageName" to "dev.icerock"),
             files = listOf(buildGradleFile, sourceCodeFile),
+            includes = listOf("src/test/resources/includes", "src/test/resources/second-includes"),
```
Comment (Alex009) [inline shaper-core/src/test/kotlin/dev/icerock/tools/shaper/core/ShaperTest.kt]: why we can't use here `listOf("includes", "second-includes")`?

Comment (ilya-shaydullin) [reply shaper-core/src/test/kotlin/dev/icerock/tools/shaper/core/ShaperTest.kt]: Fixed it

**Review by Alex009 — APPROVED** [review]

**Pull request merged**